Framework/Kernel/inc/MantidKernel/ConfigService.h

```diff
@@ -206,7 +206,7 @@ class MANTID_KERNEL_DLL ConfigServiceImpl final {
   /// Sets the log level priority for all log channels
   void setLogLevel(int logLevel, bool quiet = false);
   /// Sets the log level priority for all log channels
-  void setLogLevel(std::string logLevel, bool quiet = false);
+  void setLogLevel(std::string const &logLevel, bool quiet = false);
```

Comment:
```suggestion
  void setLogLevel(const std::string &logLevel, bool quiet = false);
```

Reply:
I prefer the read-right-to-left approach, so it's "reference to constant string", instead of the constant-first approach.  Unless that's an issue of mantid code standardization.